atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractGenericStreamStore.java

```diff
@@ -156,7 +156,7 @@ private File loadToNewTempFile(Transaction transaction, ID id, StreamMetadata me
         }
     }
 
-    private void checkStreamStored(ID id, StreamMetadata metadata) {
+    protected void checkStreamStored(ID id, StreamMetadata metadata) {
```

Comment:
When you override a method in generated code, it would be good to add a comment saying that changing this method could affect the generated code in unexpected ways.